src/runtime/wasm_math.ll

```diff
@@ -164,13 +164,7 @@ define weak_odr <8 x i16> @saturating_narrow_i32x8_to_u16x8(<8 x i32> %x) nounwi
 ;   }
 
 ; single to double-precision floating point
-
-declare <2 x double> @llvm.wasm.promote.low(<4 x float>)
-
 define weak_odr <4 x double> @float_to_double(<4 x float> %x) nounwind alwaysinline {
```

Comment:
Do we need this helper function at all now? This is probably the default CodeGen_LLVM behavior.

Reply:
Good question. Stand by.

Reply:
Nope: removing this scalarizes the operation.